tests/lints/pragma-unroll.rs

```diff
@@ -0,0 +1,47 @@
+//! Tests for the `pragma-unroll` lint.
+
+use indoc::indoc;
+
+use pretty_assertions::assert_eq;
+
+use crate::util::lint_report;
+
+#[test]
+fn basic() {
+    let code = indoc! { r#"
+        #include <linux/bpf.h>
+        #include <bpf/bpf_helpers.h>
+
+        SEC("xdp")
+        int xdp_prog(struct xdp_md *ctx)
+        {
+            void *data_end = (void *)(long)ctx->data_end;
+            void *data = (void *)(long)ctx->data;
+
+            __u32 sum = 0;
+
+            #pragma unroll
+            for (int i = 0; i < 10; i++) {
+                sum += i * 2;
+            }
+
+            if (data + sum > data_end)
+                return XDP_DROP;
+
+            return XDP_PASS;
+        }
+
+        char _license[] SEC("license") = "GPL";
```

Comment:
For testing purposes we don't really need a fully functional BPF program. While it's not wrong either, I feel like if it's too long it just distracts from the relevant bits. So at least the `_license` definition should probably go to shorten it. What do you think?

Reply:
Fine by me; I kind of liked having a "full BPF program" because to be honest I was unusre at which point tree-sitter starts parsing.

Let me take a stab at your request.

Reply:
done.